debian/control

```diff
@@ -23,9 +23,8 @@ Build-Depends: debhelper (>= 9),
 Package: fake-juju
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends},
-         juju-mongodb,
-         juju-mongodb3.2,
-         python-fakejuju
+         juju-mongodb3.2 | juju-mongodb | mongodb-org-server
+Recommends: python-fakejuju
 Description: A fake implentation of Juju using the dummy provider.
```

Comment:
I know you didn't change this, but it's jumping out at me now..

s/implentation/implementation/